documents/MIGRATION_GUIDE.md

```diff
@@ -1,70 +1,66 @@
-# Migrate from V1 to V2 of the AWS IoT SDK for Java
+# Migrate from v1 to v2 of the AWS IoT SDK for Java
 
-> [!TIP]  
-> If you can't find necessary information in this guide, the [How to Get Help](#how-to-get-help) section will guide you.
+The AWS IoT Device SDK for Java v2 is a major rewrite of the v1 SDK code base built on top of Java 8+. It includes many updates,
+such as improved consistency, ease of use, more detailed information about client status, and an offline operation queue
+control. This guide describes the major features that are new in the v2 SDK, and provides guidance on how to migrate
+your code to v2 from v1 of the AWS IoT SDK for Java.
 
-* [What’s New in V2 SDK](#whats-new-in-v2-sdk)
-* [How to Get Started with V2 SDK](#how-to-get-started-with-v2-sdk)
+> [!NOTE]  
+> If you can't find the information you need in this guide, visit the [How to get help](#how-to-get-help) section for more help and guidance.
+
+* [What's new in v2 SDK](#whats-new-in-v2-sdk)
```

Comment:
This title and links are not conformant with the new title

Reply:
I'll fix them after merging this PR, it'll be easier